### PR TITLE
Update DG Part 1

### PR DESCRIPTION
**Pull request by Eugene-Ong-W-X**

### CODE REVIEW

**Comment by florentianayuwono** [comment]
Hi, I think these changes are already covered in #125 haha

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -170,23 +170,28 @@ This section describes some noteworthy details on how certain features are imple
...
 
 These operations sort in reverse order when `isReverse` is true.
 
 These operations are exposed in the `Model` interface under the same method name.
 
 Given below is an example usage scenario and how the sorting mechanism behaves at each step.
 
-Step 1. The user executes `sort t/!friend n/` command to perform a multi-level sort. `SortCommandParser` calls `ArgumentTokenizer#tokenizeToList()` to separate the parameters of `t/!friend` and `n/`.
+Step 1. The user enters `sort t/!friend n/` command to perform a multi-level sort. `SortCommandParser` checks the user input to confirm
+that parameters have been entered. `SortCommandParser` calls `ArgumentTokenizer#tokenizeToList()` to separate the parameters of `t/!friend` and `n/`.
```
Comment (ChongCheeKaiClarence) [inline docs/DeveloperGuide.md]:
```suggestion
Step 1. The user enters `sort t/!friend n/` command to perform a multi-level sort. `SortCommandParser` checks the user input to confirm
that the parameters have been entered. `SortCommandParser` calls `ArgumentTokenizer#tokenizeToList()` to separate the parameters of `t/!friend` and `n/`.
```

**Comment by Eugene-Ong-W-X** [comment]
> Hi, I think these changes are already covered in #125 haha

Yea i know, I needed to update those lines too. And a merge conflict might erase your changes if I'm not careful 😨 so I brought some of your changes over first 😄

**Review by ChongCheeKaiClarence — COMMENTED** [review]
Great Work! Here are some suggestions.

**Review by ChongCheeKaiClarence — APPROVED** [review]
LGTM!

**Pull request merged**